shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -355,31 +363,34 @@ private void createDstDir(AlluxioURI dstPath) throws AlluxioException, IOExcepti
    * Copies a file or directory specified by srcPath from the local filesystem to dstPath in the
    * Alluxio filesystem space.
    *
-   * @param srcFile the source file in the local filesystem
+   * @param srcPath the {@link AlluxioURI} of the source in the local filesystem
    * @param dstPath the {@link AlluxioURI} of the destination
    * @throws AlluxioException when Alluxio exception occurs
    * @throws IOException when non-Alluxio exception occurs
    */
-  private void copyFromLocal(File srcFile, AlluxioURI dstPath)
+  private void copyFromLocal(AlluxioURI srcPath, AlluxioURI dstPath)
```

Comment:
Because i want to get the scheme what user input and output the path with scheme.